### ISSUE TITLE
Image download from the site 

### ISSUE THREAD

**Issue by maisonmdsgreen** [notepad-plus-plus/notepad-plus-plus#10563]
The image download from the website, could have the transparent background to match the night mode of the website.

https://notepad-plus-plus.org/downloads/images/folder_download_4.png

![site](https://user-images.githubusercontent.com/11423362/133890116-ea73f84a-cd39-487d-b15e-113b73bb9850.jpg)

Thank

**Comment by maisonmdsgreen** [comment]
And logo BitCoin

https://notepad-plus-plus.org/assets/images/bitcoinLogo.png

**Comment by CennoxX** [comment]
While we're at it, the image of Notepad++ could also change to dark theme
![grafik](https://user-images.githubusercontent.com/6975881/134133850-f4bf93d3-1e9d-4286-9e8b-3566b7ce4fc1.png)

**Comment by thomassmith311** [comment]
> @thomassmith311 You're right. All that's needed is the `background-color:#000;`:
> 
> ```
> <style id="inverter" media="screen">
>     html { filter: invert(100%) }
>     * { background-color: inherit }
>     img:not([src*=".svg"]), .colors, iframe, .demo-container { filter: invert(100%); background-color:#000 }
> </style>
> ```

Right, replacing lines 37-41 with that on the HTML file fixes the issue. I tried downloading the HTML file myself and editing it, and this was the result:
![image](https://user-images.githubusercontent.com/19561856/136600322-e34abf3d-6d57-4ae0-a22e-1271d9f8bd20.png)

**Comment by thomassmith311** [comment]
@donho could we implement this?
